package.json

```diff
@@ -26,5 +26,5 @@
     "turbo": "^1.9.3",
     "typescript": "5.1.0-beta"
   },
-  "packageManager": "pnpm@7.27.0"
+  "packageManager": "pnpm@8.5.1"
```

Comment:
should prevent a bunch of big diffs in the lockfile keeping this sync - we were already using lockfile version 6 so should probably use pnpm 8 too